packages/block-editor/src/components/colors-gradients/style.scss

```diff
@@ -1,11 +1,11 @@
 .block-editor-color-gradient-control {
 	&__color-indicator {
-		margin-bottom: $grid-size;
+		margin-bottom: $grid-unit;
```

Comment:
I don't think we should ever actually use $grid-unit, it should always be one of the numbered ones. I.e. in this case, it should be $grid-unit-10.

In that vein, $grid-unit is intended only to seed all the numbered variables. Should it have a different name to reflect that?

Reply:
I updated but why not just remove the variable in that case and use $grid-unit-10 everywhere.

Reply:
Oh I guess that could work? Seems almost cyclical 😅

The alternative would be to spell out 8px for each variable.